rooms/views.py

```diff
@@ -148,4 +211,342 @@ def get(self, request):
 
             return JsonResponse({'rooms' : result}, status=200)
         except IndexError:
-            return JsonResponse({'result' : 'INVALID GEOCODE'}, status=400)
\ No newline at end of file
+            return JsonResponse({'MESSAGE' : 'INVALID GEOCODE'}, status=400)
+        except ValueError:
+            return JsonResponse({'MESSAGE' : 'INVALID REQUEST'}, status=400)
+
+class Review(View):
+    ### 리뷰 보기 ###
+    @PublicUserDeco
+    def get(self, request):
```

Comment:
예를 들어서 room_id가 1번인 room의 리뷰 데이터 목록을 호출하는 엔드포인트라면,
RESTful하게 구현하려면, URI가 `/room/1/reviews?search=...` 형태가 되야 합니다.

resource(room data) 하나를 가져올 경우에는 `path parameter`를 사용해주세요!!
```suggestion
    def get(self, request, room_id):
        ...
```